### PR TITLE
[monkey-runner] Apply command plugin by default

### PR DESCRIPTION
**Pull request by takecare**
We can avoid having the users' of this plugin having to apply the android command plugin on their own, thus making it simpler to integrate this plugin in their projects.

Tests updated.

### CODE REVIEW

File: monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy [hunk 1]
```diff
@@ -11,7 +11,11 @@ class MonkeyConfigurationPlugin implements Plugin<Project> {
     @Override
     void apply(Project project) {
         ensureAndroidPluginAppliedTo(project)
-        ensureCommandPluginAppliedTo(project)
+
+        project.configure(project) {
```
Comment (tasomaniac) [inline monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy]: Can't we just do
`project.apply plugin: 'com.novoda.android-command'`

`apply` is a method in `Project` class.

File: monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy [hunk 2]
```diff
@@ -11,7 +11,11 @@ class MonkeyConfigurationPlugin implements Plugin<Project> {
...
+            //noinspection GroovyAssignabilityCheck
+            apply plugin: 'com.novoda.android-command'
```
Comment (takecare) [inline monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy]: we could do something like
```
project.configure(project, {
            project.plugins.apply(AndroidCommandPlugin.class)
        })
```
instead (it solves the warning)
@mr-archano rfc

**Comment by tasomaniac** [comment]
I think `README` should also be adjusted with respect to the changes here

File: monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy (hunk 1 above)
Comment (takecare) [inline monkey-runner/plugin/src/main/groovy/com/novoda/monkey/MonkeyConfigurationPlugin.groovy]: yes we can! 👍

**Comment by takecare** [comment]
@tasomaniac true, i always forget

**Comment by takecare** [comment]
@tasomaniac i understand that point of view and spent some time thinking it over before suggesting that change. my reasoning is:

1. the command plugin is already a dependency either way and its tasks will always be bundled in;
2. requiring the user (dev) to apply it does make it obvious but that's it;
3. if we are clear enough about the dependency i think it works out in the end.

anyway, more than open to other people's suggestions/opinions

**Comment by tasomaniac** [comment]
Makes sense. Thanks for the explanation. 
Regarding to point 3, we should maybe still have some words about command plugin in the Readme. What do you think?

**Comment by takecare** [comment]
@tasomaniac haven't forgot about it! hopefully will be changing the readme today

**Pull request merged**